toxav/bwcontroller.c

```diff
@@ -52,31 +54,32 @@ struct BWController_s {
 
     uint32_t packet_loss_counted_cycles;
     Mono_Time *bwc_mono_time;
+    bool bwc_receive_active;
```

Comment:
What does this do? Can you add a comment to say what it means for this to be true or false?